chart/compass/charts/gateway/templates/deployment.yaml

```diff
@@ -101,6 +108,45 @@ spec:
                   name: {{ .Values.global.auditlog.configMapName }}
                   key: auditlog-oauth-tenant-var
                   optional: true
+            {{ else if eq .Values.gateway.auditlog.authMode "oauth-mtls" }}
+            - name: APP_AUDITLOG_AUTH_MODE
+              value: "oauth-mtls"
+            - name: APP_AUDITLOG_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.global.auditlog.secret.name }}
+                  key: {{ .Values.global.auditlog.secret.clientIdKey }}
+            - name: APP_AUDITLOG_SKIP_SSL_VALIDATION
+              value: "{{ .Values.global.auditlog.skipSSLValidation }}"
```

Comment:
maybe it's better this to be false because this is productive configuration? 
P.S. There is same property for the tests I left comment there